aspnet/fundamentals/logging.rst

```diff
@@ -143,7 +143,7 @@ First, be sure to add the ``Microsoft.Framework.Logging.TraceSource`` package to
 	:lines: 5-12
 	:emphasize-lines: 8
 
-The following example demonstrates how to configure two separate ``TraceSourceLogger``s for an application, both logging only ``Critical`` messages. Each call to ``AddTraceSource`` takes a ``TraceListener``. The first call configures a ``ConsoleTraceListener``; the second one configures an ``EventLogTraceListener`` to write to the ``Application`` event log. These two listeners are not available in DNX Core, so their configuration is wrapped in a conditional compilation statement (``#if DNX451``).
+The following example demonstrates how to configure two separate ``TraceSourceLoggers`` for an application, both logging only ``Critical`` messages. Each call to ``AddTraceSource`` takes a ``TraceListener``. The first call configures a ``ConsoleTraceListener``; the second one configures an ``EventLogTraceListener`` to write to the ``Application`` event log. These two listeners are not available in DNX Core, so their configuration is wrapped in a conditional compilation statement (``#if DNX451``).
```

Comment:
I completely agree with you regarding consistency. My vote would be not to pluralize a class name, but instead to use another word (e.g., instances) to indicate plurality.